### DISCUSSION TITLE
Language resources in other languages

### DISCUSSION THREAD

**Discussion by delightedbydhamma** [delightedbydhamma/impossible-pali-project#3] [General]
Migrating some stuff from our discourse conversation over here.

**Gillian said**
On the matter of Latin, I had a chat with [@Gralock](https://discourse.suttacentral.net/u/gralock) today. He drew my attention to this conversational Latin course

Here is one of his videos. if you to go his channels you can see loads of stuff at different levels, though i think to get all the course material you have to buy a package. i reluctantly have to resist adding Latin to the languages i am dabbling in. [Greetings in Latin · Lingua Latina Comprehensibilis 1A · Latin Language Lessons, Beginner Latin - YouTube](https://youtu.be/j7hd799IznU)
And this one has transcription and translation [Spoken Latin: Latin Greetings - Latin dialogue and conversation - YouTube](https://youtu.be/BbcRm5EbGxg)
I also love his time traveler video. i can imagine a video of a time traveler going back to india at the time of the buddha and chatting with one of hif followers in Pali. [Time traveler meets Ancient Roman soldier! Interview with the Legionary ⚔️ (subtitles available) - YouTube](https://youtu.be/M2ULzPUcaxU)On the matter of Latin, I had a chat with [@Gralock](https://discourse.suttacentral.net/u/gralock) today. He drew my attention to this conversational Latin course

**Comment by GillianPerrett** [comment]
I find this material excellent cos students are shown by the social context what the phrases mean and then ditto where  the inflections (case endings on nouns and pronouns belong). They van turn on captions for transcription and translation if they wish. So the student has a choice of learning by just looking and listening, or by reading and listening. Or both of course. However a dyslexic student could avoid reading.

**Comment by GillianPerrett** [comment]
I've been down lots of different rabbit holes this arvo.  atm am lost somewhere in the warren, but to record where I've been:

**General**
TestPrepInsight  https://www.youtube.com/playlist?list=PLqY8FxGmpUSCvMkPhEh-P2uQRoY3hxd6e. Playlist with reviews/comparisons of lots of language learning apps
Olly Richards  https://www.youtube.com/playlist?list=PLQJscr8iS4eGSdyUsDYuAV3YZXOYZJn0F  Playlist of different imersion methods.
Olly Richards. https://www.youtube.com/playlist?list=PLQJscr8iS4eE2kLl1tOtqvmdmCul6Gw2v. Playlist of how to learn a language through reading

**APPS**
Pimsleur. https://www.pimsleur.com/the-pimsleur-method here and app 
Rosetta Stone https://www.rosettastone.com here or 2 apps 
Quizlet. https://quizlet.com/en-gb  here or app
Flashcards Deluxe  http://orangeorapple.com/Flashcards/  here and app
Linq. https://www.lingq.com/en/. here and app
Memrise  https://www.memrise.com  here and app
Babbel. https://www.babbel.com  here and app
Duolingo

Acc Olly Richards there are 4 types of useful language learning APP;
<img width="452" alt="Screenshot 2023-03-19 at 3 19 15 pm" src="https://user-images.githubusercontent.com/48712748/226158311-b5bab1e5-fbf6-4d32-935d-911f0fea5e0e.png">
Of these, content and flashcards are what we want to consider. 

**Spanish - Compare:**
Duolingo
Olly Richards - https://storylearning.com, https://www.youtube.com/@storylearning/about. ?Applicable to sutta stories
SpanishDict https://www.spanishdict.com/lessons/387?lesson_start_source=grammar%20article%20start%20next%20lesson.  Feels more wholistic and faster paced than Duolingo

**Reply by delightedbydhamma** [reply]
I signed up for Linq yesterday. It looks similar to storylearning. For whatever reason I picked Romanian! 
They work on the premise of vocab and sentences only with grammar rules later.
Here's a video from the founder https://www.youtube.com/watch?v=B2zRk4PqMSA

The free version was severely limited, but after 10 minutes or so I was not really motivated to continue. It was mostly just sentences on a page with audio and then you tick off the words you know (anki style) for each sentence. The advantage of what we are doing is that people will hopefully be interested in the content and feel motivated and excited to memorise it and understand it. That being said, the mix of exercises in DuoLingo suited me better.

I couldn't get into SpanishDict without a credit card. They wanted me to sign up.

I still have songs from Pimsleur French (early 2000s) stuck in the back of my memory!

**Reply by GillianPerrett** [reply]
My experience of LinQ has been more positive.  I opted for Malay.  It wasn't sentences on a page; the first lesson was a conversation that took two pages.  Then different ways of remembering, testing, recombining the component words.  When the free section ended I was so motivated that I am wondering whether to pay and continue.  That said, I wonder if there's quite a bit of variation among languages on the same platform as I didn't find a mix of exercises in DuoLingo and was quite turned off!

I don't think Spanish Dict is different enough ... I can show it to you on my computer anyway.  It seems that I used Babbel for Spanish as well, tho I have no memory of it.   I have not yet opened Story Learning. ... hopefully keeping the best til last.

**Comment by GillianPerrett** [comment]
I have continued my study of Malay and found that LingQ does have significant detriments, ie
-  Very buggy (eg played audio for different text from that displayed, claims to work on iOS but jammed)
-  Poor security (I saw a gate crasher advertising crypto currency, th)e Safari extension carries a security warning 
-  Quality of texts for first lessons poor
-  Too much learner control, eg of assigning English meanings, gaging level of texts
-   Dictionary look up is about as good as SC’s - ie not adequate for beginners 

But these are all things, we can avoid, and **the design concept is very good.**  For starters:
-  Each lesson is based on text
-  Learners can import any text that interests them
-  The lessons automatically break to words, phrases and sentences
-  Audio for all 4 levels
-  Learners have choice over which of the 4 levels, + written or audio, to start with, and can flip between them at will
-  It provides learner control of what to do with new vocabulary items, in both the short and long term
-  Learners can install dictionaries
-  There a system of reward and encouragement (in the form of 'coins', which = our % bar)
[I may be back to add to this list.]

These, I think are very strong features, tho I don't think that learners being able to import is relevant to what we are doing atm.  In terms of starting from text and allowing learner flexibility around levels of language and listening and reading, it's the best basic model than any I've looked at yet, a strong core around which additional features can be built.  (Just got Story Learning to go.)

**Comment by delightedbydhamma** [comment]
Here are some screenshots from Duolingo showing the different styles of exercises.
1.)
![image](https://user-images.githubusercontent.com/105905254/234726774-9b6ef875-bc7a-4c67-9cdc-c299fed37fba.png)
2.)
![image](https://user-images.githubusercontent.com/105905254/234726843-ee447f8b-d040-4517-90aa-536573f48762.png)
3.)
![image](https://user-images.githubusercontent.com/105905254/234726896-a89df987-a55c-443f-a8f0-7fb22ab5c355.png)
4.)
![image](https://user-images.githubusercontent.com/105905254/234726926-4859b91e-443f-45c8-ae68-ed7c9e6f3780.png)
5.)
![image](https://user-images.githubusercontent.com/105905254/234726988-8e801e5d-1e2d-4645-8871-7d009a93e5b8.png)
6.)
![image](https://user-images.githubusercontent.com/105905254/234727036-5625129f-ee3b-4d51-b098-f7293e678ed8.png)
7.)
![image](https://user-images.githubusercontent.com/105905254/234727107-b4e8ef32-8fd3-4f5b-967a-7d6e690fd3c7.png)
8.)
![image](https://user-images.githubusercontent.com/105905254/234727189-feda9813-26b1-43af-9f4f-f703506e060e.png)
9.)
![image](https://user-images.githubusercontent.com/105905254/234727245-182705f6-fd90-4126-b886-3d3f984bbd86.png)
10.)
![image](https://user-images.githubusercontent.com/105905254/234727290-cfb51f42-21dc-4f03-abeb-aeffce655a69.png)

**Reply by GillianPerrett** [reply]
Great!  Was just about to remind you.  I started analysing each activity but found I can't see where one screen shot ends and the next begins.  Sorry to be a pain, but would you please put a space above each screenshot, and number them sequentially.  Then I can match my comments to the right picture.

**Reply by delightedbydhamma** [reply]
Ok renumbered. Not easy to format on my iPad

**Comment by GillianPerrett** [comment]
![image](https://user-images.githubusercontent.com/48712748/234794003-986a52bc-fbe6-4670-a7d5-5d90e5dcb26c.png)
This is the sort of way teachers think when they're keeping the linguistic description at the back of their minds.

**Comment by GillianPerrett** [comment]
Here are some additional screenshots from Duolingo:

11.   **Choose the picture** (listen TL, read TL, choose from pictures)- ....  This one is OK, unlike No.6.

![4607598A-FDD1-43D0-8216-773E526B4BE3_1_201_a](https://user-images.githubusercontent.com/48712748/235089367-a6bcc36a-cb49-4b5b-bc54-44c97bc71191.jpeg)

12.   **Tap what you hear** (listen TL, read TL, choose) 

![30BC0252-0F44-4D51-B58A-E61C951B2F68_1_201_a](https://user-images.githubusercontent.com/48712748/235095661-5dc15737-8d53-4c54-8424-c5c75c51182b.jpeg)

13.  **Speak the correct response** (read TL, listen TL, speak TL, choose TL)

![F23E03A1-F421-4B11-B476-31A920F640A3_1_201_a](https://user-images.githubusercontent.com/48712748/235089874-65d9b57b-8cac-4f93-af20-9cca8d7c0d0e.jpeg)

14.  **Speak this sentence** (Listen TL, read TL, speak TL)

![F856524E-CB2B-42E1-81FA-F5ADD02F2179_1_201_a](https://user-images.githubusercontent.com/48712748/235094929-79c20dd1-dc09-452e-8af0-44a862665713.jpeg)

15.  **Tap what you hear** (Listen TL, read TL, choose phoneme)

![AB711C37-5D61-4955-A275-3AAA666B8E0F_1_201_a](https://user-images.githubusercontent.com/48712748/235096354-f9d4251d-8789-4148-94b1-f601967fc2f1.jpeg)

16.  **What do you hear** (listen TL, read TL, choose from among minimal pairs)

![D9C35561-420F-47E0-BD7F-482D395770BA_1_201_a](https://user-images.githubusercontent.com/48712748/235096923-6f4b1c98-26dd-4c83-aed1-1900305682e2.jpeg)

17.  **Incentivation screen**

![361F2E58-4F16-4E32-997D-8CA06044348A_1_201_a](https://user-images.githubusercontent.com/48712748/235097098-0d4febcf-bb28-42fa-99e6-ceb7a9aa8543.jpeg)

**Reply by anni-anni** [reply]
Might be debatable but in my opinion, for people possessing a higher percentage of "Linguistic Intelligence" - it is far easier and more beneficial to earn from natural sources like native speaking videos. 
Simply download a fun/interesting video (in the language you desire to learn) and upload it into a [video translator](https://wavel.ai/solutions/ai-video-translator/) then compare and learn OR learn through old school subtitles by using a subtitle generator AI...

This process worked out for me, hence recommending ...

**Reply by GillianPerrett** [reply]
Well Pali is what is technically (and rather rudely) called a dead language.  It has no native speakers.
& I don't think Bollywood has made a Pali movie yet!
But there is a point here, one way forward would be to start compiling those rare single voice recordings in Pali by "nonnative" speakers.  I think we could do that.
